_topic_map.yml

```diff
@@ -933,3 +933,49 @@ Topics:
   Topics:
   - Name: Container-native Virtualization 2.0 release notes
     File: cnv-release-notes
+#---
+#Name: Serverless applications
+#Dir: serverless
+#Distros: openshift-enterprise
+#Topics:
+#- Name: Serverless overview
+#  File: serverless-overview
+#---
+#Name: Container-native Virtualization
```

Comment:
Will you check your TOC to make sure that you're just changing the serverless section? I'm not sure how you picked up service mesh and CNV changes, but we don't want to drop this content from the TOC.

Reply:
There were two instances in my file, maybe caused by a rebase or something.
This one at the bottom seems to have been a placeholder, I checked which is the right one from the current repo and removed the 'placeholder' versions.